.github/workflows/e2e-test.yaml

```
@@ -6,6 +6,7 @@ on:
     paths-ignore: # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-ignoring-paths
       - 'docs/**'
       - '.atlassian/**'
+      - 'test/**'
```

```suggestion
      - 'test/unittest**'
```

If we have `test/**` it means the e2e test won't run if there is a change to the e2e test (which we want).